Sources/KlaviyoSwift/KlaviyoAPI.swift

```diff
@@ -96,11 +97,11 @@ extension KlaviyoAPI.KlaviyoRequest {
     func encodeBody() throws -> Data {
         switch self.endpoint {
         case .createProfile(let payload):
-            return try environment.analytics.encodeJSON(payload)
```

Comment:
Theme here is we need explicit types for older versions of swift. AnyEncodable should encode the same way as before. The other option here would be to create specific encode method for each type which might have been tedious.